test/test-meshctl-ls.json

```diff
@@ -0,0 +1,1388 @@
+{
```

Comment:
are you comparing against this? because its likely to change as any deps change. might be better to assert just on some fields known to be stable.